### PR TITLE
XWIKI-20758: Administration menu violates tablist structure

### PR DESCRIPTION
**Pull request by Sereza7**
## Jira
https://jira.xwiki.org/browse/XWIKI-20758
## PR Changes
* Removed the inappropriate roles from the accordion
* Removed the 'aria-multiselectable' attribute that conveyed wrong information to assistive technology users.

## Notes
See the comments on the jira ticket for explanations on why this solution was chosen.

## View
20758-afterPR
There is no longer 'role does not contain children' error reported by axe-core.

### CODE REVIEW

File: xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml [hunk 1]
```diff
@@ -302,11 +302,11 @@
     #set ($children = [])
     #sortCollectionOfMapsByField($item.children, 'order', 99999, 'asc', $children)
     &lt;div class="panel panel-default"&gt;
-      &lt;a class="panel-heading#if (!$isActive) collapsed#end" role="tab" id="panel-heading-$escapedId"
+      &lt;a class="panel-heading#if (!$isActive) collapsed#end" id="panel-heading-$escapedId"
         href="$!item.url" data-toggle="collapse"#if ("$!options.id" != '') data-parent="#$options.id" #end
         data-target="#panel-body-$escapedId" aria-expanded="$isActive" aria-controls="panel-body-$escapedId"
         title="$!escapetool.xml($item.description)"&gt;$!services.icon.renderHTML($item.icon)$escapetool.xml($name)&lt;/a&gt;
-      &lt;div class="panel-collapse collapse#if ($isActive) in#end" role="tabpanel" id="panel-body-$escapedId"
+      &lt;div role='region' class="panel-collapse collapse#if ($isActive) in#end" id="panel-body-$escapedId"
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml]: I was following the [pattern for accordion provided by WAI-ARIA](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/)
However, after reading the [MDN doc about region](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/region_role), it seems like the navigation role is more appropriate for our use case.
Addressed in https://github.com/xwiki/xwiki-platform/pull/2284/commits/b8d56b17a4818efed4061ebdcd151920ba1579ba

File: xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml [hunk 2]
```diff
@@ -270,7 +270,7 @@
 
 #macro (verticalNavigation $menu $options)
   {{html clean="false"}}
-  &lt;div id="$!options.id" class="panel-group $!options.cssClass" role="tablist" aria-multiselectable="true"&gt;
+  &lt;div id="$!options.id" class="panel-group $!options.cssClass"&gt;
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml]: If we already set all the visible elements as 'navigation', we don't need to use navigation here.
In combination with the changes described in https://github.com/xwiki/xwiki-platform/pull/2284#discussion_r1290174001, I think it makes the most sense to use a \<section> here (native HTML for role=region).
This way, it's still quite easy for screen reader access and understand that an extra step has to be made before navigation is unlocked, namely 'to select the category of administration'.

20758-landmarksFixed
(on this screenshot, I didn't update the translation file so the value for the region label is not right)

Addressed in https://github.com/xwiki/xwiki-platform/pull/2284/commits/b8d56b17a4818efed4061ebdcd151920ba1579ba too.

[the remark this answers is not in the document]
Comment (michitux) [reply xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml]: We have many navigation elements where parts of the navigation are hidden and only expand when you click on some button/link, e.g., breadcrumbs or navigation trees. This is quite similar to such a menu imho. Therefore, I would consider this whole left thing as a navigation. On the other hand, having a navigation labelled "Mail" is imho not clear at all. Consider also the "Search" navigation which is very close to the main search navigation and could thus confuse users.

[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminSheet.xml]: :+1:  Addressed in https://github.com/xwiki/xwiki-platform/pull/2284/commits/e5741267d4126a932ee01ec1b7e8184d4c28a715

**Comment by Sereza7** [comment]
Addressed all comments so far.

File: xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminTranslations.xml [hunk 3]
```diff
@@ -44,6 +44,7 @@ administration.sectionTitle=Administration: {0}
 administration.sectionTitle.page=Page Administration: {0}
 administration.sectionTitle.wiki=Wiki Administration: {0}
 administration.sectionTitle.global=Global Administration: {0}
+administration.menu.label=Administration category picker.
```
[the remark this answers is not in the document]
Comment (Sereza7) [reply xwiki-platform-core/xwiki-platform-administration/xwiki-platform-administration-ui/src/main/resources/XWiki/AdminTranslations.xml]: :+1: new value: "Administration sections"
Addressed in https://github.com/xwiki/xwiki-platform/pull/2284/commits/c3561d84fae4802a74669c5bd1773b33d4e2ea61

**Pull request merged**